src/components/title-with-description/TitleWithDescription.tsx

```diff
@@ -15,31 +14,37 @@ interface TitleWithDescriptionProps {
     title?: SxProps
     description?: SxProps
   }
+  useTooltip?: boolean
 }
 
 const TitleWithDescription = ({
   title,
   description,
-  style = styles
+  style = styles,
+  useTooltip = false
 }: TitleWithDescriptionProps) => {
   const [tooltipVisible, setTooltipVisible] = useState<boolean>(false)
 
   const handleTooltip = () => setTooltipVisible((prevState) => !prevState)
-
-  return (
-    <Box sx={style.wrapper}>
-      <Typography sx={style.title}>{title}</Typography>
-      <Tooltip open={tooltipVisible} placement='bottom' title={description}>
-        <Typography
-          component={ComponentEnum.Span}
-          onClick={handleTooltip}
-          sx={style.description}
-        >
-          {description}
-        </Typography>
-      </Tooltip>
-    </Box>
-  )
+  if (useTooltip) {
+    return (
+      <Box sx={style.wrapper}>
+        <Typography sx={style.title}>{title}</Typography>
+        <Tooltip open={tooltipVisible} placement='bottom' title={description}>
+          <Typography onClick={handleTooltip} sx={style.description}>
+            {description}
+          </Typography>
+        </Tooltip>
+      </Box>
+    )
+  } else {
+    return (
+      <Box sx={style.wrapper}>
+        <Typography sx={style.title}>{title}</Typography>
+        <Typography sx={style.description}>{description}</Typography>
+      </Box>
+    )
+  }
```

Comment:
i think you can do it with less code, maybe something like this:
```js
    <Box sx={style.wrapper}>
      <Typography sx={style.title}>{title}</Typography>
      {useTooltip && (
        <Tooltip open={tooltipVisible} placement='bottom' title={description}>
          <Typography onClick={handleTooltip} sx={style.description}>
            {description}
          </Typography>
        </Tooltip>
      )}
    </Box>
  ```

Reply:
fixed